modules/internal/DefaultSparse.chpl

```diff
@@ -269,6 +293,28 @@ module DefaultSparse {
       for e in data[1..dom.nnz] do yield e;
     }
 
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
+      const numElems = dom.nnz;
+      const numChunks = _computeNumChunks(numElems): numElems.type;
+      if debugDefaultSparse {
+        writeln("DefaultSparseArr standalone: ", numChunks, " chunks, ",
+                numElems, " elems");
+      }
+      if numChunks <= 1 {
+        for i in 1..numElems {
+          yield data[i];
+        }
+      } else {
+        coforall chunk in 1..numChunks {
+          const (startIx, endIx) =
+            _computeChunkStartEnd(numElems, numChunks, chunk);
+          for e in data[startIx..endIx] do
+            yield e;
```

Comment:
Is there a reason the `numChunks <= 1` case does 

```chapel 
for i in 1..numElems do 
  yield data[i];
```

but this one does: 

```chapel
for e in data[startIx..endIx] do 
  yield e;
```

It seems like they should either either both iterate over the index, or both iterate over the elements of the slice?